client/src/store/slices/cartSlice/cartSlice.js

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable no-underscore-dangle */
 /* eslint-disable no-return-assign */
 /* eslint-disable no-param-reassign */
 import { createSlice } from '@reduxjs/toolkit';
 import axiosConfig from '../../../axiosConfig';
+import getProduct from '../../../api/getProduct';
+import updateCart from '../../../api/updateCart';
```

Comment:
if we use this functions only once and only in cartSlice, you can incapsulate it here

Reply:
encapsulation is ok as long as it doesnt go against clear readable code. 
in slices. we only write reducer functions and thunk functions which dispatch actions. Any other API logic is written in api folder in order to maintain a clean slice file using the modular approach.